xpdacq/calib.py

```diff
@@ -147,6 +147,14 @@ def run_calibration(exposure=5, dark_sub_bool=True,
                                     detector=detector,
                                     calibrant=calibrant
                                     )
+    print("INFO: Please navigate to the analysis terminal to complete "
```

Comment:
We should put a post it note on the analysis computer, saying "analysis computer" :smiling_imp: